src/app/features/request/new/new.component.html

```diff
@@ -1,5 +1,169 @@
 <app-page>
   <ng-container content>
+    <div class="header">
+      <div class="container md:container md:mx-auto p-16 pt-44">
+        <h1 class="font-bold text-white text-6xl pb-8">طلب عرض سعر..</h1>
+        <p
+          class="font-light text-white text-2xl leading-10 max-w-screen-lg pb-20"
+        >
+          هناك حقيقة مثبتة منذ زمن طويل وهي أن المحتوى المقروء لصفحة ما سيلهي
+          القارئ عن التركيز على الشكل الخارجي للنص أو شكل توضع الفقرات في الصفحة
+          التي يقرأها
+        </p>
+      </div>
+    </div>
+    <div class="body relative">
+      <div
+        class="container md:container md:mx-auto bg-white p-16 pt-4 absolute start-0 -translate-y-24"
```

Comment:
"md:container" class is useless in case "container" exists 

remove "md:container"


classes "absolute, start-0" are useless
Don't forget to remove relative class from parent element